pom.xml

```diff
@@ -118,58 +118,51 @@
       <subscribe>user-subscribe@commons.apache.org</subscribe>
       <unsubscribe>user-unsubscribe@commons.apache.org</unsubscribe>
       <post>user@commons.apache.org</post>
-      <archive>http://mail-archives.apache.org/mod_mbox/commons-user/</archive>
+      <archive>https://mail-archives.apache.org/mod_mbox/commons-user/</archive>
```

Comment:
Wouldn't be better to use https://lists.apache.org/list.html?user@commons.apache.org instead? It's a more user-friendly solution.

Reply:
I was unaware of that option. Will note for the future.